CODING_GUIDELINES.md

```diff
@@ -207,7 +207,36 @@ class MyClass
 }
 ```
 
-### 8. Misc.###
+### 8. Header inclusion order.###
+The headers should be placed in the following order:
+ 1. Module header (in .cpp)
+ 2. System/Qt/Boost etc. headers (splitted in subcategories if you have many).
```

Comment:
>May one split and order them by module in the following way

IMO, it will be too much. I never saw the point in using these prefixes (Qt module names).